releases.yml

```diff
@@ -26,34 +27,22 @@ releases:
       rhcos:
```

Comment:
Don't we generally comment this out when overriding `reference_releases!: {} `=

Reply:
Ah, maybe. But existence of `assembly.basis.assembly` and fiddling with builds should be enough hint to reconstruct what is going on, imho.
But maybe it is worth to write some place "why are we doing this weird thing". Guess `issues.include` possibly provides that?